it/ApiConnectorISpec.scala

```diff
@@ -16,11 +17,32 @@ class ApiConnectorISpec extends BaseISpec {
 
       editSetResponse mustBe editSets.editSet1
     }
+  }
 
-    s"get the edit set 100 times for id: $idOfExistingEditSet" in {
-      val action = (1 to 100).toList.traverse(_ => apiConnector.getEditSet(idOfExistingEditSet))
-
-      action.unsafeRunSync().foreach(_ mustBe editSets.editSet1)
+  "when a request for an editSetRecord is made" must {
```

Comment:
Here's a suggested alternative which reduces some duplication:

```
    Set(
      "COAL.2022.V1RJW.P",
      "COAL.2022.V2RJW",
      "COAL.2022.V3RJW.P",
      "COAL.2022.V4RJW.P",
      "COAL.2022.V5RJW.P",
      "COAL.2022.V6RJW.P",
      "COAL.2022.V7RJW.P",
      "COAL.2022.V8RJW.P",
      "COAL.2022.V9RJW.P",
      "COAL.2022.V10RJW.P",
      "COAL.2022.V11RJW.P",
      "COAL.2022.V12RJW.P"
    ).foreach { oci =>
      s"get an edit set record for oci: $oci" in {
        val editSetRecordResponse =
          apiConnector.getEditSetRecord(idOfExistingEditSet, oci).unsafeRunSync()

        editSetRecordResponse mustBe editSetRecords.getEditSetRecordByOCI(oci)
      }
    }
```

Reply:
Yeah, that works, too. I didn't think of using a table like that, but it reduce the duplication.

Small typo on line 47 (`editSetRecordRespose`).